From 6a736fcf5786080c2d85f9feccc3269cf7124190 Mon Sep 17 00:00:00 2001
From: Francesco Conti <fconti@amazon.com>
Date: Mon, 15 Nov 2021 15:05:51 +0100
Subject: [PATCH 1/2] Do not load certificates from default paths on MacOS

Since the root certificates are already loaded it is not
necessary to load the certificates from the defualt paths
---
 tls/openssl/gtlsdatabase-openssl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tls/openssl/gtlsdatabase-openssl.c b/tls/openssl/gtlsdatabase-openssl.c
index 7ba7b8c..65709fc 100644
--- a/tls/openssl/gtlsdatabase-openssl.c
+++ b/tls/openssl/gtlsdatabase-openssl.c
@@ -221,8 +221,7 @@ g_tls_database_openssl_populate_trust_list (GTlsDatabaseOpenssl  *self,
     }
 
   CFRelease (anchors);
-#endif
-
+#else
   if (!X509_STORE_set_default_paths (store))
     {
       char error_buffer[256];
@@ -232,6 +231,7 @@ g_tls_database_openssl_populate_trust_list (GTlsDatabaseOpenssl  *self,
                    error_buffer);
       return FALSE;
     }
+#endif
 
   return TRUE;
 }
-- 
2.25.1

